### PR TITLE
(maint) Update README

### PR DESCRIPTION
**Pull request by mwaggett**
This commit updates the README to set expectations for contributing.

### CODE REVIEW

**Review by Magisus — APPROVED** [review]

**Review by donoghuc — APPROVED** [review]

**Comment by donoghuc** [comment]
Disclaimer, i did not run orch tests with clj-parent based on this PR :trollface:

File: README.md [hunk 1]
```diff
@@ -6,12 +6,25 @@ Good candidates for things to include here:
...
+## Contributing
+
+All updates should be made via pull request (PR).
+
+Updates should include a CHANGELOG entry and commit message with a brief explanation of what is being updated and why.
+
+#### Approving PRs
+In order to give meaningful approval to a PR, you must either run your component's test suite against the changes or confirm that your component does not use the dependencies being updated. **Please still approve the PR even if you aren't using the dependencies in question.**
+
+If your teammate opened the PR, you may assume that they ran the appropriate tests and you may approve as a sanity check.
```
Comment (steveax) [inline README.md:18]: Not sure that sentence is needed, but if so, can we use something other than ‘sanity’? I’m kinda fond of ‘Spock’.

Comment (mwaggett) [reply README.md:18]: ah yeah, thanks for calling that out! I will update.

I guess the reason I added this sentence was to imply that someone from the author's team should still look at the PR, but I can definitely make that clearer.

Comment (mwaggett) [reply README.md]: updated!

**Review by lazartea — APPROVED** [review]

**Comment by jonathannewman** [comment]
One observation - clj-parent is a public repo and the README seems internally focused.  How do we expect external contributors to participate?  Do we?

**Review by rbrw — APPROVED** [review]

**Review by Zak-Kent — APPROVED** [review]

**Comment by mwaggett** [comment]
@jonathannewman I don't know if we _expect_ external contributions, but I think we are open to them (I just don't think I've seen any in my time here, at least as far as I can remember). The `Approving PRs` section does mention teams, but I kinda feel like it's all still applicable to external contributors - they should still submit PRs and need approval from all teams. Did you have any particular changes in mind to make things more general to external folks?

**Comment by rbrw** [comment]
One minor commment - I'd typically hard wrap the lines so that it might be easier to read from a variety of editors/pagers, and so that it may work better with git's defaults (with respect to diffing/merging/etc.), but no strong feelings if the consensus is for unwrapped lines here.

**Comment by jonathannewman** [comment]
@mwaggett I was just thinking it would not be possible for external contributors to follow those steps related to releasing the package.  Perhaps stating that if the contributor is a Puppet employee, they are expected to be responsible to release the package, and if the contributor is not a puppet employee, it is expected that a Puppet employee will take responsibility for releasing the package for the contributor.

**Comment by jonathannewman** [comment]
Or just add a disclaimer to the release section that Puppet employees will be responsible for this part of the process?

**Comment by mwaggett** [comment]
ahh yeah, fair enough. I'll add a note to the `Releases` section. thanks!

**Comment by mwaggett** [comment]
okay @jonathannewman note added!

and @rbrw I added line breaks at sentence boundaries, I hope that helps.

**Comment by jonathannewman** [comment]
Thanks for doing this, it is a big improvement.

LGTM! Fantastic!

**Pull request merged**